status-im/status-react
# chore: add docs with size

Added docs for recent decision made by theme on how to handle sizes in the code base

To test:
This pr updates the use of this new api for a set of components. Nothing should change.

these components include:
Group avatar
Icon avatar

Network dropdown

Preview list

System message

Settings/ Data item

Context tag

Network tags

Number tags

Wallet keypair
